### PR TITLE
Fix pr-129 conflicts

### PR DESCRIPTION
**Pull request by johnbrett**

### CODE REVIEW

**Comment by johnbrett** [comment]
This will close https://github.com/johnbrett/hapi-auth-bearer-token/issues/124.

Actually, let me fix these whitespace issues before reviewing this PR.

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,5 @@ node_modules
 
 npm-debug.log
 coverage.html
+
+.vscode/settings.json
```
Comment (AdriVanHoudt) [inline .gitignore]: why not the whole folder?

**Comment by AdriVanHoudt** [comment]
Wait is this for https://github.com/johnbrett/hapi-auth-bearer-token/pull/129 or #124 ?

**Comment by johnbrett** [comment]
this PR fell down due to my inactivity - my bad @nakardo. Are you still looking for this feature? We have just released a new version of this module with support for v17, so will have to redo this PR to fit that, but I'm happy to take that on if you're still looking for it.

**Comment by nakardo** [comment]
@johnbrett No problem. I currently don't own anymore the project I was working on -- I left it working in the meantime with the forked version. So, feel free to close this one.

**Comment by lbvo** [comment]
Hi,
First of all thanks a lot for this plugin, it was a perfect match for me!
However I am trying to implement another strategy in to my api, where a certain route should be able to be configured with 2 strategies, where first would be allowed via query if fail fallback and verify authentication via other strategy in this occasion it would be a Bearer in the header. 
I am having trouble in doing this, that was when I found this PR, it seems highly related to my issue (yes?)

I would very much like to stay with this model for authentication, I have tried a few and this is the best of the bunch I have been working with.

What is the conclusion for this, it seems it was never merged to master ?

-- Have nice day, regards Lasse.

**Comment by johnbrett** [comment]
@lbvo this PR went stale unfortunately (likely my fault), I think a fresh PR will have to be made against master fixing this problem. I don't currently have time to create one myself, but would happily support you in making one to fix this problem if you're willing to?

**Comment by nakardo** [comment]
@lbvo @johnbrett if I have time I'll try to see if it's easy to port back the functionality from the old PR into the Hapi v17 code.

**Comment by johnbrett** [comment]
That would be awesome @nakardo!!

**Pull request closed without merging**